test/e2e/util.go

```diff
@@ -444,11 +444,14 @@ func DeleteRC(c *client.Client, ns, name string) error {
 	return nil
 }
 
-// Launch a Replication Controller and wait for all pods it spawns
-// to become running. The controller will need to be cleaned up external
-// to this method
+// RunRC Launches (and verifies correctness) of a Replication Controller
+// It will waits for all pods it spawns to become "Running".
+// The controller will need to be cleaned up external to this method.
```

Comment:
nit: Last line would read better as "It's the caller's responsibility to clean up the controller."